recipes-ni/initscripts-nilrt/files/mountconfig

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+# Copyright (c) 2012-2013 National Instruments.
+# All rights reserved.
+
+partition_label="boot-config"
+volume_label="config"
+volume_number="3"
+mount_point="/etc/natinst/share"
+mount_args="-o sync"
+RESTORECON="/sbin/restorecon"
+
+if [ -e /etc/init.d/mountutils ] ; then
+	# include common utilities
+	. /etc/init.d/mountutils
+fi
+
+# Bail out on first error
+set -o errexit
+
+mount_config()
+{
+	# Don't mount if already mounted
+	if ! grep -q " $mount_point " /proc/self/mounts; then
+		if cat /sys/class/mtd/mtd*/name 2>/dev/null | grep -q ^$partition_label\$
+		then
+			# If an mtd partition has this partition label, use
+			# mount_ubi_volume
+			mount_ubi_volume "$partition_label" "$volume_label" \
+				"$volume_number" "$mount_point" "$mount_args"
```

Comment:
The `config` partition only exists on ARM targets, iirc. So I don't think this initscript is relevant to dunfell images and beyond. It can be dropped.